pinot-clients/pinot-java-client/src/main/java/org/apache/pinot/client/BrokerCacheUpdaterManual.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.pinot.client;
+
+import java.util.List;
+
+
+/**
+ * Controller based broker cache that only supports manual refresh
+ */
+public class BrokerCacheUpdaterManual implements UpdatableBrokerCache {
+  private final BrokerCache _brokerCache;
+
+  public BrokerCacheUpdaterManual(String scheme, String controllerHost, int controllerPort) {
+    _brokerCache = new BrokerCache(scheme, controllerHost, controllerPort);
+  }
+
+  public void init() throws Exception { }
```

Comment:
(code style) This does not follow `Pinot Style`
```suggestion
  public void init()
      throws Exception {
  }
```

Reply:
Ack